student/src/main/java/tds/student/services/ExamCompletionServiceImpl.java

```diff
@@ -44,12 +44,13 @@ public ReviewTestServiceImpl(@Qualifier("integrationOpportunityService") final I
     }
 
     @Override
-    public ResponseData<String> reviewTest(final TestOpportunity testOpportunity, final TestManager testManager)
+    public ResponseData<String> updateStatusWithValidation(final TestOpportunity testOpportunity, final TestManager testManager,
+                                                           final String statusCode)
         throws ReturnStatusException {
         ResponseData<String> responseData = new ResponseData<>(TDSReplyCode.OK.getCode(), "OK", null);
 
         if (isLegacyCallsEnabled) {
-            responseData = legacyReviewTest(testOpportunity, testManager);
+            responseData = legacyReviewTest(testOpportunity, testManager, statusCode);
```

Comment:
Actually, status doesn't matter.  I could pass in Start status and legacy would pass.